rootless-install.sh

```diff
@@ -327,7 +338,21 @@ print_instructions() {
 
 	echo "export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/docker.sock"
 	echo
-	service_instructions
+	v1903_service_instructions
+}
+
+exec_setuptool() {
+	args="$@"
+	if [ -n "$FORCE_ROOTLESS_INSTALL" ]; then
+		args="$args --force"
+	fi
+	if [ -n "$SKIP_IPTABLES" ]; then
+		args="$args --skip-iptables"
+	fi
+	set -x
+	(
+		"$BIN/dockerd-rootless-setuptool.sh" install $args
+	)
```

Comment:
Instead of this extra `args` variable, I think we should just use `set`, especially since it will properly preserve any whitespace, which isn't relevant right now but could be in the future (also, the `set -x` should be inside the subshell or it will leak and the subshell won't do us any good :sweat_smile:):

```suggestion
	if [ -n "$FORCE_ROOTLESS_INSTALL" ]; then
		set -- "$@" --force
	fi
	if [ -n "$SKIP_IPTABLES" ]; then
		set -- "$@" --skip-iptables
	fi
	(
		set -x
		"$BIN/dockerd-rootless-setuptool.sh" install "$@"
	)
```

Reply:
thanks, updated